src/lib/global_configuration.c

```diff
@@ -68,13 +68,13 @@ bool libreport_load_global_configuration_from_dirs(const char *dirs[], int dir_f
 
         map_string_iter_t iter;
         g_hash_table_iter_init(&iter, s_global_settings);
-        const char *key, *value;
-        while(libreport_next_map_string_iter(&iter, &key, &value))
+        gpointer key, value;
+        while(g_hash_table_iter_next(&iter, &key, &value))
```

Comment:
`value` is not used so it can be `NULL`.

Reply:
What I meant was that you could call `g_hash_table_iter_next(&iter, &key, NULL)`.